experiments/memcached-sensitivity-profile/docs/run_experiment.md

```diff
@@ -19,153 +19,134 @@
-Following the 4 threads, the configuration below is configured to reach 800 concurrent connections to memcached (the calculations are provided in [Tuning Mutilate](tuning.md#Mutilate-Tuning) section).
 
-```bash
-## --- aggressors ---
-export SWAN_AGGR=l1d,l1i,stream,membw,caffe
-
-## --- isolation ---
-export SWAN_HP_CPUS=4
-export SWAN_BE_CPUS=4
-
-## --- memcached configuration ---
-export SWAN_MEMCACHED_PATH=$GOPATH/src/github.com/intelsdi-x/swan/workloads/data_caching/memcached/memcached-1.4.25/build/memcached
-export SWAN_MEMCACHED_IP=192.168.10.9
-export SWAN_MEMCACHED_THREADS=4
-export SWAN_MEMCACHED_CONNECTIONS=16000
-
-## --- mutilate configuration ---
-# master
-export SWAN_MUTILATE_PATH=/usr/local/bin/mutilate
-export SWAN_MUTILATE_MASTER=192.168.10.10
-export SWAN_MUTILATE_MASTER_THREADS=4
-export SWAN_MUTILATE_MASTER_CONNECTIONS=4
-export SWAN_MUTILATE_MASTER_CONNECTIONS_DEPTH=4
-export SWAN_MUTILATE_MASTER_BLOCKING=true
-export SWAN_MUTILATE_MASTER_QPS=1000
-export SWAN_MUTILATE_WARMUP_TIME=5s
-
-# agent
-export SWAN_MUTILATE_AGENT=192.168.10.3,192.168.10.4,192.168.10.5,192.168.10.6
-export SWAN_MUTILATE_AGENT_THREADS=20
-export SWAN_MUTILATE_AGENT_CONNECTIONS=10
-export SWAN_MUTILATE_AGENT_CONNECTIONS_DEPTH=4
-export SWAN_MUTILATE_AGENT_BLOCKING=true
-export SWAN_MUTILATE_AGENT_AFFINITY=true
-export SWAN_MUTILATE_AGENT_PORT=6556
-
-## --- experiment configuration ---
-export SWAN_SLO=500
-export SWAN_LOAD_DURATION=15s
-export SWAN_PEAK_LOAD=650000
-export SWAN_LOAD_POINTS=10
-export SWAN_REPS=1
-export SWAN_LOG=error
-
-## --- snap configuration
-export SWAN_SNAPTELD_ADDRESS=192.168.10.9
-
-## --- cassandra configuration
-export SWAN_CASSANDRA_ADDR=192.168.10.10
-export SWAN_SNAP_CASSANDRA_PLUGIN_PATH=$GOPATH/bin/snap-plugin-publisher-cassandra
+In this example, SUT node has 32 hyper threads over 16 physical cores on 2 sockets. Per the topology description showed in [Theory](theory.md) section, this leaves 4 threads and logical cores for memcached.
+ 
+Please paste following snippet into `config.ini` file.
+Some variables should be changed before running the experiment.
+
+1. `REMOTE_SSH_LOGIN` and `REMOTE_SSH_KEY_PATH` should point to user and key that is authorized to SSH on every machine in the experiment cluster.
+1. `MEMCACHED_USER` should be the same as `REMOTE_SSH_LOGIN`
+1. `MEMCACHED_LISTENING_ADDRESS` should be SUT node address.
+1. `EXPERIMENT_PEAK_LOAD` should be provided appropriate to SUT machine. If tiny VM is used, it might be even as low as 100000. If high-end server machine is used, it might be as high as millions requests per second.
+1. `EXPERIMENT_SLO` should be set appropriate to SUT machine (in microseconds). In VM environment, the default value should be increased 10x. 
+1. When SUT node has HyperThreads, `EXPERIMENT_BE_WORKLOADS` should contain `stress-ng-cache-l1` entry to examine Memcached sensitivity to L1 cache interference. 
```

Comment:
I don't think I follow. Do we want to prove that running two processes on single cpu-thread makes interference? I'm not sure if this was the goal of L1 aggressor.